Makefile

```diff
@@ -148,26 +148,26 @@ lint-install:
 
 .PHONY: lint
 lint:
-ifeq (, $(shell which golangci-lint))
-	$(info golangci-lint can't be found, please run: make lint-install)
-	exit 1
-endif
+	ifeq (, $(shell which golangci-lint))
+		$(info golangci-lint can't be found, please run: make lint-install)
+		exit 1
+	endif
 
 	golangci-lint run
 
-ifeq (, $(shell which markdownlint-cli))
-	$(info markdownlint-cli can't be found, please run: make lint-install)
-	exit 1
-endif
+	ifeq (, $(shell which markdownlint))
+		$(info markdownlint can't be found, please run: make lint-install)
+		exit 1
+	endif
 
 	markdownlint-cli
```

Comment:
Isn't that contradictory? It checks for `markdownlint` but then runs `markdownlint-cli`.

Reply:
Ah yes, I need to change it in one more place. I was focused on the odd indented sections. Good catch